From e4e7df80d5a0c49dcf48a686e75284d9b299d401 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Fri, 31 Aug 2018 13:13:30 +0530
Subject: [PATCH 6/6] Fix pkg-config file library list

-lm was duplicated, and -lpthread should never be used. It should be
-pthread.
---
 libs.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libs.mk b/libs.mk
index 9a6092a..7fbea4c 100644
--- a/libs.mk
+++ b/libs.mk
@@ -331,9 +331,9 @@ vpx.pc: config.mk libs.mk
 	$(qexec)echo 'Conflicts:' >> $@
 	$(qexec)echo 'Libs: -L$${libdir} -lvpx -lm' >> $@
 ifeq ($(HAVE_PTHREAD_H),yes)
-	$(qexec)echo 'Libs.private: -lm -lpthread' >> $@
+	$(qexec)echo 'Libs.private: -pthread' >> $@
 else
-	$(qexec)echo 'Libs.private: -lm' >> $@
+	$(qexec)echo 'Libs.private: ' >> $@
 endif
 	$(qexec)echo 'Cflags: -I$${includedir}' >> $@
 INSTALL-LIBS-yes += $(LIBSUBDIR)/pkgconfig/vpx.pc
-- 
2.12.2.windows.2

